PSIni/Functions/Get-IniContent.ps1

```diff
@@ -141,7 +141,15 @@ Function Get-IniContent {
                 }
                 $name, $value = $matches[1..2]
                 Write-Verbose "$($MyInvocation.MyCommand.Name):: Adding key $name with value: $value"
-                $ini[$section][$name] = $value
+                if ($ini[$section][$name] -is [string]) {
+                    $ini[$section][$name] = $ini[$section][$name], $value
```

Comment:
Why are you appending as array, if it's a string?

Reply:
This is the strategy. If the content is only one value, then it is a simple string. If a second value needs to be added, the entry is converted to an array. If you have a better idea how to handle this, please let me know.

Reply:
after tinkering a bit with this, I would like you to replace the code with this:

```powershell
   if ( -not $ini[$section][$name]) {
        $ini[$section][$name] = @($value)
    }
    else {
        $ini[$section][$name] += $value
    }
```

Reply:
I have done as you suggested.